dbms/src/Storages/StorageDisaggregatedRemote.cpp

```diff
@@ -507,17 +507,25 @@ DM::Remote::RNWorkersPtr StorageDisaggregated::buildRNWorkers(
         log);
     const auto read_mode = DM::DeltaMergeStore::getReadMode(db_context, table_scan.isFastScan(), table_scan.keepOrder(), push_down_filter);
     const UInt64 read_tso = sender_target_mpp_task_id.query_id.start_ts;
-    LOG_DEBUG(log, "Building segment input streams, read_mode={} is_fast_scan={} keep_order={}", magic_enum::enum_name(read_mode), table_scan.isFastScan(), table_scan.keepOrder());
-
-    return DM::Remote::RNWorkers::create({
-        .log = log->getChild(executor_id),
-        .read_task = read_task,
-        .columns_to_read = column_defines,
-        .read_tso = read_tso,
-        .push_down_filter = push_down_filter,
-        .read_mode = read_mode,
-        .cluster = db_context.getTMTContext().getKVCluster(),
-    });
+    LOG_DEBUG(
```

Comment:
```suggestion
    LOG_INFO(
```
I think using info level here is better